console/src/locales/index.ts

```diff
@@ -25,6 +26,15 @@ export const locales = [
     code: "zh",
     package: zhCN,
   },
+  {
+    name: "正體中文",
+    code: "zh-TW",
+    package: zhTW,
+  },
+  {
+    code: "zh",
+    package: zhTW,
+  },  
```

Comment:
Those lines could be removed safely, because we have defined the default locale above.

```suggestion
  }
```

Reply:
Thanks for your correction🙏